src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ResponseDeadlineExtensionClaimantNotificationHandler.java

```diff
@@ -61,8 +65,8 @@ private CallbackResponse notifyClaimantForDeadlineExtension(CallbackParams callb
         CaseData caseData = callbackParams.getCaseData();
 
         notificationService.sendMail(
-            caseData.getApplicantSolicitor1UserDetails().getEmail(),
-            notificationsProperties.getClaimantDeadlineExtension(),
+            addEmail(caseData),
```

Comment:
would name it getEmail(caseData) because you are getting an email not adding it. and getTemplate

Reply:
method name updated